Add contribution statement and checking (#585)
MetOffice/fab: CONTRIBUTORS.md

# Contributor List

If this is your first contribution, you will need to agree to the

> [MOMENTUM Open Source Software Contributor Agreement](https://github.com/MetOffice/Momentum/blob/main/CLA.md)

Reviewer: ```suggestion
[Momentum Open Source Software Contributor Agreement](https://github.com/MetOffice/Momentum/blob/main/CLA.md)
```

. A copy of v1.0 is provided below for cenvienience but the version linked

> above always takes precidence.

Reviewer: ```suggestion
. A copy of v1.0 is provided [below](#momentum-open-source-software-contributor-agreement) for convenience but the version linked
above always takes precedence.
```

To indicate your agreement, add your details to the the following table.

If you are not submitting contributions on behalf of an organisation please use
"n/a" for your affiliation.

| GitHub Username | Real Name       | Affiliation |

> |-----------------|-----------------|-------------|
> | MatthewHambley  | Matthew Hambley | Met Office  |
> 
> 
> ## Momentum Open Source Software Contributor Agreement
> 

Reviewer: ```suggestion
| MatthewHambley  | Matthew Hambley | Met Office  |

---

## Momentum Open Source Software Contributor Agreement

```

# Contributing to Fab

Anyone is welcome to contribute to the "Fab" project, provided they conduct
themselves in good faith and with respect for all.

## Support Requests

If you need help with something, please make use of the "Discussions" tab on
the GitHub project page. Do not jump straight to raising an issue, it may be
that your problem is easily answered by the community and does not require
changes to the code.

## Raising Bug Reports and Feature Requests

Bug reports and feature requests may be raised using the "Issues" tab, but
before you do, please check there isn't a similar closed issue. Also check
the "Discussion" tab to see if someone has already come across what you have
found.

## Contributions

Contributions may be made by raising a pull request against the trunk branch
at <https://github.com/MetOffice/fab>. You will need a personal fork of the
repository to host your development branch.

If this is your first contribution, your branch will need to include the

> addition your details to the the following list. If you are not submitting

Reviewer 2: ```suggestion
addition of your details to the the following list. If you are not submitting
```

contributions on behalf of an organisation please use "n/a" for your
affiliation.

+-----------------+-----------------+-------------+
| GitHub Username | Real Name       | Affiliation |
+=================+=================+=============+
| MatthewHambley  | Matthew Hambley | Met Office  |
+-----------------+-----------------+-------------+

## Contributor Licence Agreement and Certificate of Origin

By making a contribution to this project, I certify that:

1. The contribution was created in whole or in part by me and I have the right
   to submit it, either on my behalf or on behalf of my employer, under the
   terms and conditions as described by this file; or
1. The contribution is based upon previous work that, to the best of my
   knowledge, is covered under an appropriate licence and I have the right or
   permission from the copyright owner under that licence to submit that work
   with modifications, whether created in whole or in part by me, under the
   terms and conditions as described by this file; or
1. The contribution was provided directly to me by some other person who

>    certified (a) or (b) and I have not modified it.

Reviewer 2: ```suggestion
   certified (1) or (2) and I have not modified it.
```
